src/components/sectionList/SectionListWrapper.tsx

```diff
@@ -24,19 +25,52 @@ type SectionListWrapperProps = {
     data: ModelCollection<BaseListModel> | undefined
     pager: Pager | undefined
     error: FetchError | undefined
+    refetch: () => void
 }
 
 export const SectionListWrapper = ({
     data,
     error,
     pager,
+    refetch,
 }: SectionListWrapperProps) => {
     const { columns: headerColumns } = useModelListView()
     const schema = useSchemaFromHandle()
     const { selectedModels, checkAllSelected, add, remove, toggle, clearAll } =
         useSelectedModels()
     const [detailsId, setDetailsId] = useState<string | undefined>()
     const [sharingDialogId, setSharingDialogId] = useState<string | undefined>()
+    const { show: showDeletionSuccess } = useAlert(
+        i18n.t('Successfully deleted the {{model}}', {
+            model: schema.singular,
+        }),
+        { success: true }
+    )
+    const { show: showDeletionFailure } = useAlert(
+        i18n.t('Failed deleting the {{model}}!', {
```

Comment:
I think `Failed to delete` sounds better.

Reply:
Reworded